HangMan/HangMan2.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Apr  9 15:39:01 2020
+
+@author: cpantarotto
+"""
+# Lib
+import random
+#import numpy
+# DataSet of words
+global_dictionary = ["outside","squirrel", "straight", 
+              "finger", "leaf", "frog", "mouth", "grandson"]
+def hangmanGraphic (numberOfLives):
+    if numberOfLives==6:
+        print("_________")
+        print("|/   | ")
+        print("|")
+        print("|")
+        print("|")
+        print("|")
+        print("|")
+        print("|_______")            
+    if numberOfLives==5:
```

Comment:
you can use the keywork `elif` to optimize this function

Reply:
Done